internal/config/profile.go

```diff
@@ -227,6 +227,12 @@ func (p *profile) SetOrgID(v string) {
 	p.Set(orgID, v)
 }
 
+// Exists returns true if there are any set settings for the profile name.
+func Exists(name string) bool { return p.Exists(name) }
+func (p *profile) Exists(name string) bool {
+	return len(viper.GetStringMap(name)) == 0
```

Comment:
similar to above, shouldn't this be 
```suggestion
	return len(viper.GetStringMap(name)) > 0
```

To have semantical sense?